cmd/node/config/api.toml

```diff
@@ -93,3 +93,8 @@
          # /transaction/:txhash will return the transaction in JSON format based on its hash
          { Name = "/:txhash", Open = true },
 	]
+
+[APIPackages.history]
+	Routes = [
+	    { Name = "/transaction/:txhash", Open = false },
+	]
```

Comment:
Regarding this comment from Binance:
`if you have blocks,please also provide: getLatestBlockNum,getBlockByHeight,getBlockByHash,getTxsByBlockHeight,getTxsByBlockHash`

We already have info about the latest block number in `network/status` (this is not very intuitive since this info has nothing in common with `network`. I don't know if it makes sense for refactoring here NOW since we have clients already using this routes, but we need to keep this in mind :) ). 

BUT we need a `[APIPackages.block]` route group. Following our rest standard we can have `block/:blockHash` (which will get info from the header I assume). Seems that they also need to fetch blocks by height, so we can also have `block/by-nonce/:nonce`.

It also seems they need `address/transactions` with pagination. I don't think this is feasible to get from the nodes, should be discussed :) - I think this should be resolved with the elastic api

Reply:
`address/transactions` seems to be not required (given our Telegram conversations with Binance).